src/signals_notebook/entities/entity.py

```diff
@@ -187,3 +187,6 @@ def get_html(self) -> str:
         log.info('Html template for %s:%s has been rendered.', self.__class__.__name__, self.eid)
 
         return template.render(data=data)
+
+    def dump(self, *args, **kwargs):
```

Comment:
Why signature differs from 
```
def dump(self, base_path: str, fs_handler: FSHandler):
```
?
Mypy will punish you

Reply:
There is a chance it will vary depending on item type. I dont want code to crash if we try to dump some entity which is not implemented yet